### DISCUSSION TITLE
Balanced portfolio settings

### DISCUSSION THREAD

**Discussion by seguidor777** [rodrigo-brito/ninjabot#65] [Q&A] [answered]
Hi all,

I'm writing a ninjabot client and I want to handle a balanced portfolio as in this project [1]. I saw that we can pass a pairs list to the ninjabot settings, but we cannot specify the weight for each asset. Are you interested on implementing this feature? if so, I could help with the implementation, I have good experience in Go. If not, I'll have to work in a separate fork to make use of this feature.

Thanks,
Jorge

[1] https://github.com/kdmukai/binance_bbb#customize-your-portfolio

**Comment by rodrigo-brito** [comment, marked as answer]
Hi @seguidor777, the Ninjabot allows the developer to control the size of each order. Then you can set the weight in your strategy and create orders to balance the position in a period of time. I will create an example for you. Just a minute.

**Comment by rodrigo-brito** [comment]
@seguidor777 check this example: https://github.com/rodrigo-brito/ninjabot-portfolio-balancer
I defined a wallet with four coins 20% for each and 20% of free cash.

**Reply by seguidor777** [reply]
Awesome, thanks man

**Reply by berisberis** [reply]
Awesome! This is what I was looking for. Thanks!

**Comment by seguidor777** [comment]
With the help of your example, I have created a similar strategy that also uses asset weights to diversify whenever the wallet has a minimum balance or more and it never sells. I left the repository link here for anyone who is interested
https://github.com/seguidor777/portfel

**Reply by rodrigo-brito** [reply]
Great job!
